packages/components/src/components/alert/readme.md

```diff
@@ -67,6 +67,15 @@ Die **Alert**-Komponente wurde auf die Darstellung mit höchstmöglichen Kontras
 
 Das zusätzliche Icon (je nach gewähltem Typ) gewährleistet, dass die Information vom Nutzer nicht allein über die Farbe klassifiziert werden muss.
 
+Bei der Nutzung eines Screenreaders wird beim Lesen des Icons zusätzlich der Titel mitgelesen.
+Der Titel sollte daher nicht mit dem **Alert**-Typ beginnen, da das zu einem doppelten Vorlesen des **Alert**-Typs führt.
+<br>
+Beispiel:
+<br>
+Richtig: Label = "Füllen Sie das Feld aus" -> Ausgabe: Fehler - Füllen Sie das Feld aus
+<br>
+Falsch: Label = "Fehler: Füllen Sie das Feld aus." -> Ausgabe: Fehler - Fehler: Füllen ie das Feld aus
```

Comment:
Typo:

```suggestion
Falsch: Label = "Fehler: Füllen Sie das Feld aus." -> Ausgabe: Fehler - Fehler: Füllen Sie das Feld aus
```

Außerdem sind die `<br>` in Markdown nicht notwendig, würde ich durch Absätze ersetzen oder wo gewollt mit der MD-Syntax (`  ` - zwei Leerzeichen) einfügen.